README.md

```diff
@@ -73,7 +73,7 @@ _**Note**_: *For this you need to install git on your machine. You can download
 
    `git clone https://github.com/anitab-org/stem-diverse-tv`
 
- * Now after you cloned the repository, get into the anitab-org.github.io directory by -
+ * Now after you cloned the repository, get into the stem-diverse-tv directory by -
```

Comment:
get into doesn't seem correct. Could you change it?

Reply:
Changed it to "move to"